apps/grgsm_livemon_headless.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
```

Comment:
Remove `grgsm_livemon_headless.py` from the repository as it is generated/compiled from `grgsm_livemon_headless.grc` during build.

Reply:
Solved.